build.mk

```diff
@@ -3,6 +3,7 @@
 LUCY_PRESENT=$(shell if [[ -e "lucy" ]]; then echo "true"; else echo "false"; fi)
 
 image:
+	docker pull quay.io/jupyter/scipy-notebook:latest
```

Comment:
now would be a good time to swap out using the "latest" tag to a more stable tag.  I'd suggest the date-based tag, which currently appears to be "2024-02-26"

Reply:
I don't think using the dated releases is appropriate because if you check the Quay repository, they're supposed to be the same thing.  I'm OKi with doing `:latest` for now and see how it behaves.

Reply:
See latest commit.  Happy?   :)